src/custom/hooks/useSwapCallback.ts

```diff
@@ -1,18 +1,21 @@
 import { SwapCallbackState } from '@src/hooks/useSwapCallback'
-import { INITIAL_ALLOWED_SLIPPAGE } from 'constants/index'
+import { BIPS_BASE, INITIAL_ALLOWED_SLIPPAGE } from 'constants/index'
 
-// import { useSwapCallback as useSwapCallbackUniswap } from '@src/hooks/useSwapCallback'
 import { BigNumber } from 'ethers'
-import { Percent, Trade, TradeType } from '@uniswap/sdk'
 
+import { ETHER, Percent, Trade, TradeType } from '@uniswap/sdk'
 import { useActiveWeb3React } from '@src/hooks'
 import useENS from '@src/hooks/useENS'
 import { useMemo } from 'react'
 import useTransactionDeadline from '@src/hooks/useTransactionDeadline'
+
+import { computeSlippageAdjustedAmounts } from '@src/utils/prices'
```

Comment:
i don't think we need every import here with `@src` prepended, i think

Reply:
Ok.

Why do we have `@src` at all, BTW? Does it have to do with the custom path thingy?

Reply:
So, I removed all `@src/` import prefixes and it didn't quite workout.
Got some weird errors regarding import order on `useContracts`...

Anyway, imports look quite messy on this file.
I'll try to sort them out.